lib/Net/SAML2/Protocol/Artifact.pm

```diff
@@ -0,0 +1,145 @@
+use strict;
+use warnings;
+package Net::SAML2::Protocol::Artifact;
+# VERSION
+
+use Moose;
+use MooseX::Types::DateTime qw/ DateTime /;
+use MooseX::Types::Common::String qw/ NonEmptySimpleStr /;
```

Comment:
We don't use this NonEmptySimpleStr in this module.
Same for use DateTime and DateTime::HiRes

Reply:
Thanks, I did go through and eliminate some (copied from other module) I will review